src/hooks/useEnableToken.ts

```diff
@@ -33,9 +33,10 @@ export const useEnableTokens = (params: Params): Result => {
     assert(!enabled, 'The token was already enabled')
 
     setEnabling(true)
+
     // TODO: Review after implementing connect wallet.
     //   Probably some APIs should have an implicit user and it should be login aware
-    walletApi.connect()
+    // walletApi.connect()
```

Comment:
Remove this entirely?

Reply:
There's a comment regarding this in the description of the PR.

I just created a issue for this #105